packages/functional-tests/tests/subscription-tests/coupons-ui-tests/resubscription.spec.ts

```diff
@@ -148,7 +148,7 @@ test.describe('resubscription test', () => {
     pages: { relier, subscribe, login, settings, subscriptionManagement },
   }, { project }) => {
     test.skip(
-      project.name === 'production',
+      project.name !== 'local',
```

Comment:
@ashrivastava-qa Looks like the test is still being run. It thought we wanted to disable this for all envrionments.

Reply:
Yes sorry, pushed again!